### PR TITLE
Drop quay.io

### LINKED ISSUE #125: Use local images instead of quay.io by default
The default should be:

* local images
* no upload to anywhere
* add option to change that (basically we have that, maybe add a function for pushing?)
* update README

### PR DESCRIPTION
**Pull request by jajik**
close #125

### CODE REVIEW

**Review by rhusar — CHANGES_REQUESTED** [review]
Looks good but the names are off.

I would drop the "-local" suffix, that's unnecessary. Also, any image can be made not local by pushing it remotely. Also "-image" is unnecessary - the parameter for docker run is an image name. But more importantly, the names are not unique enough - think about what these are for - they are for testing mod_proxy_cluster and they are products of the testsuite. So how about "mod_proxy_cluster-testsuite-tomcat-10.0" and analogous.

File: test/README.md [hunk 1]
```diff
@@ -34,8 +34,8 @@ repository.
 
 Also, if you use `podman` instead of `docker`, make sure you have `podman-docker` package installed (tests are using `docker`).
 
-If you don't want to use quay.io (the default), just set `IMG` and `HTTPD_IMG` variables to docker, local repository or some
-other service.
+By default local `httpd-local` and `tomcat-local` images are used. You can control the names by setting `IMG` and `HTTPD_IMG`
```
Comment (rhusar) [inline test/README.md:37]: Drop the -local.

File: test/httpd/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Build the image
 ```bash
-docker build -t quay.io/${USER}/mod_cluster_httpd .
+docker build -t httpd-image .
```
Comment (rhusar) [inline test/httpd/README.md:3]: Drop the -image suffice.

File: test/testsuite.sh [hunk 3]
```diff
@@ -12,10 +12,10 @@ if [ -z ${ITERATION_COUNT+x} ]; then
     export ITERATION_COUNT=50 # the number of iteration of starting/stopping a tomcat
 fi
 if [ -z ${IMG+x} ]; then
-    export IMG=quay.io/$USER/tomcat_mod_cluster
+    export IMG=tomcat-local
 fi
 if [ -z ${HTTPD_IMG+x} ]; then
-    export HTTPD_IMG=quay.io/${USER}/httpd_mod_cluster
+    export HTTPD_IMG=httpd-local
```
Comment (rhusar) [inline test/testsuite.sh:18]: And here.

**Comment by jajik** [comment]
Point taken. Let's change that if we ever get to that point. :)

**Comment by rhusar** [comment]
> Point taken. Let's change that if we ever get to that point. :)

👍🏼  Agreed.

**Comment by rhusar** [comment]
Merged, thanks @jajik

**Pull request merged**